File: Venmo/src/main/java/com/braintreepayments/api/VenmoPaymentAuthRequestCallback.java
package com.braintreepayments.api;

import androidx.annotation.NonNull;

/**
 * Callback to handle result from
 * {@link VenmoClient#tokenize(VenmoPaymentAuthResultInfo, VenmoTokenizeCallback)}
 */
public interface VenmoPaymentAuthRequestCallback {
    void onVenmoPaymentAuthRequest(@NonNull VenmoPaymentAuthRequest venmoPaymentAuthRequest);

}

File: Venmo/src/main/java/com/braintreepayments/api/venmo/VenmoPaymentAuthRequestCallback.kt
package com.braintreepayments.api.venmo

/**
 * Callback to handle result from [VenmoClient.createPaymentAuthRequest].
 */
fun interface VenmoPaymentAuthRequestCallback {

    fun onVenmoPaymentAuthRequest(venmoPaymentAuthRequest: VenmoPaymentAuthRequest)
}
